src/app/core/admin/setup-wizard/setup-wizard-config.ts

```diff
@@ -17,88 +17,3 @@ export interface SetupWizardStep {
   text: string;
   actions?: MenuItem[];
 }
-
-export const defaultSetupWizardConfig: SetupWizardConfig = {
```

Comment:
TODO: switch demo to German / French and copy the texts into json files for assets under education_de + education_fr